Source/WebKit/UIProcess/AuxiliaryProcessProxy.h

```diff
@@ -277,6 +278,19 @@ class AuxiliaryProcessProxy
     static RefPtr<WebCore::SharedBuffer> fetchAudioComponentServerRegistrations();
 #endif
 
+    virtual void didFinishProcessInitialization()
+    {
+        WebCore::Timer::schedule(1_s, [weakThis = WeakPtr { this }]() {
```

Comment:
We're not supposed to be using WebCore::Timer in the UIProcess because the app may mix WebKit1 and WebKit2. RunLoop::Timer is the one that is safe to use in the UIProcess.